Loader/AbstractSwaggerConfigurationLoader.php

```diff
@@ -75,6 +75,13 @@ public function getSchemaOperationCollection(): SchemaOperationCollection
         return $this->operationCollection;
     }
 
+    /**
+     * @param string $path
+     *
+     * @return string
+     */
+    abstract protected function getRouteAlias(string $path): string;
```

Comment:
Currently this method has been implemented only in one child. So other loaders will trigger exception.
Anyway, we do not need this method - just revert this change.